cli/tests/extensions/module_loader.rs

```diff
@@ -56,23 +57,3 @@ fn module_with_non_allowed_url_fails_to_load() {
         .failure()
         .stderr(predicate::str::contains("importing from domains other than"));
 }
-
-// A symlink is directly created during the test, as no symlinks are committed
-// to the repo.
-#[cfg(unix)]
-#[test]
-fn symlinks_are_rejected() {
-    let test_cli = TestCli::builder().build();
-    let ext_path =
-        test_cli.temp_path().to_owned().join("phylum").join("extensions").join("symlink");
-
-    test_cli.install_extension(&fixtures_path().join("symlink")).success();
-
-    std::os::unix::fs::symlink(ext_path.join("symlink_me.ts"), ext_path.join("symlink.ts"))
-        .unwrap();
-
-    test_cli
-        .run(&["symlink"])
-        .failure()
-        .stderr(predicate::str::contains("importing from symlinks is not allowed"));
-}
```

Comment:
Why was this removed? If we want it removed, we should probably also remove the associated fixture.